conda_forge_tick/git_utils.py

```diff
@@ -1678,32 +1678,22 @@ def refresh_pr(
             # might as well do it.
             # This also ensure all the json will soon get the 'last_fetched' key
             # and logic can be simplified.
-            force_refresh = False
+            last_fetched: str | None = pr_json.get("last_fetched", None)
+            if last_fetched:
+                last_fetched_date: datetime = dateutil.parser.isoparse(last_fetched)
+            else:
+                last_fetched_date = datetime.now() - timedelta(
+                    days=pr_refresh_age_days + 1
+                )
```

Comment:
I get that this unifies the `if age_days > pr_refresh_age_days:`, but I find it circuitous to construct just the right value that then _fails_ the condition.